apps/src/makerlab/BoardController.js

```diff
@@ -4,6 +4,9 @@
 /* global Promise */
 
 try {
+  // Polyfill node's process.hrtime for the browser, gets used by johnny-five.
+  process.hrtime = require('browser-process-hrtime');
```

Comment:
Where does the `process` global get defined? This would break if it isn't defined right? Might want to add some code to protect against that.